R/ml_cov_search.R

```diff
@@ -88,7 +93,6 @@ ml_cov_search <- function(data, pop_param, cov_continuous, cov_factors, seed = 1
           y = y_xgb_train,
           maxRuns = 200,
           doTrace = 0,
-          seed = 42,
```

Comment:
removed seed argument specified here, it does not actually do anything and was just returning a warning that this argument is not supported